Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml

```
@@ -1,3 +1,43 @@
+- type: entityTable
+  id: FilingCabinetFillTable
+  table: !type:AllSelector
+    children:
+    - !type:GroupSelector
+      rolls: !type:RangeNumberSelector
+        range: 2, 4
+      children:
+      - id: BoxFolderBlue
+      - id: BoxFolderRed
+      - id: BoxFolderYellow
+      - id: BoxFolderWhite
+      - id: BoxFolderGrey
+      - id: BoxFolderBlack
+    - !type:GroupSelector
+      rolls: !type:RangeNumberSelector
+        range: 0, 3
+      children:
+      - id: Paper
+      - id: PaperOffice
+    - !type:GroupSelector
+      prob: .3
+      rolls: !type:ConstantNumberSelector
+        value: 2
+      children:
+      - id: Pen
+    - !type:GroupSelector
+      prob: 0.15
+      children:
+      - !type:NoneSelector
+      - !type:AllSelector
+        weight: 0.2
+        children: 
```

This feels more complicated than it needs to be and obfuscates the actual probability

```suggestion
    - !type:AllSelector
      prob: 0.025
      children:
```

---

thanks